src/app/GitUI/UserControls/FileStatusList.cs

```diff
@@ -1517,7 +1517,7 @@ private void FileStatusListView_ContextMenu_Opening(object? sender, CancelEventA
             }
 
             // Show 'Show file differences for all parents' menu item if it is possible that there are multiple first revisions
-            bool mayBeMultipleRevs = _enableDisablingShowDiffForAllParents && GitItemStatusesWithDescription.Count > 1;
+            bool mayBeMultipleRevs = _enableDisablingShowDiffForAllParents;
```

Comment:
I believe this was intentional, there were users that were very annoyed by multi select
but it is fine for me